vpr/src/route/rr_graph2.h

```diff
@@ -19,7 +19,7 @@ enum e_seg_details_type {
 typedef vtr::NdMatrix<short, 6> t_sblock_pattern;
 
 struct t_opin_connections_scratchpad {
```

Comment:
Good to comment what this is for, if you know. Why a dimension of 8?
Does it store RRNodeIds, or is it some tile structure that stores track numbers or some such (which would be better left as an int)?